tensorboard/webapp/widgets/data_table/content_cell_component.ts

```diff
@@ -61,6 +61,7 @@ export class ContentCellComponent {
       case ColumnHeaderType.STEP_AT_MIN:
       case ColumnHeaderType.MEAN:
       case ColumnHeaderType.HPARAM:
+      case ColumnHeaderType.EXPERIMENT:
```

Comment:
This is wrong. The experiment is not a number. It should be up next to the RUN type. Did this work?

Reply:
I think I misunderstood what was happening here. This did work because there is a `typeof` check one line 65. I think these two blocks could actually be merged but I'll keep them separate for now.

Reply:
Oh, I see.  I would still like this to move with RUN anyway for now as those two types should be formatted in a similar way. I guess it makes sense that since this isn't a number it just gets returned. I think I put that there to make the compiler happy and it was never supposed to get used :P. 

I hope to do a whole type refactor to have much more generic categories.